Plugins/Covfie/include/Acts/Plugins/Covfie/CovfieConverter.hpp

```diff
@@ -0,0 +1,14 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+namespace acts::covfie_conversion{
+
+    bool test();
+}
```

Comment:
You will get notified about End of line / newline - so you can start fixing the files below,